### PR TITLE
Completed string_manipulation.md

### PR DESCRIPTION
**Pull request by joylin06**
I completed the Allen Iverson and Green Eggs and Ham exercises.

### CODE REVIEW

File: joy_lin/alpha_challenges/string_manipulation.md
Comment (jmmander) [inline joy_lin/alpha_challenges/string_manipulation.md]: There's actually a cool built-in method `chr` for getting the first character of a string (which is the same as your word[0]).
I.e:
```
my_string = 'cat in the hat'
my_string.chr => 'c'
```

Comment (joylin06) [reply joy_lin/alpha_challenges/string_manipulation.md]: Wow, got it!

**Review by jmmander — APPROVED** [review]
Looks good to me!! Maybe just get one more approval before merging though 😄 @hamidoudiallo96 @narissa-hajratalli

**Review by hamidoudiallo96 — APPROVED** [review]
Nice job :thumbsup:

**Pull request merged**
